src/policy/policy.cpp

```diff
@@ -87,7 +87,7 @@ bool IsStandard(const CScript& scriptPubKey, const std::optional<unsigned>& max_
         unsigned char m = vSolutions.front()[0];
         unsigned char n = vSolutions.back()[0];
         // Support up to x-of-3 multisig txns as standard
```

Comment:
Should probably replace `// Support up to x-of-3 multisig txns as standard` with
`// Support up to x-of-MAX_BARE_MULTISIG_PUBKEYS_NUM multisig txns as standard`